File: P485 Max Consecutive Ones/Solution.java
public class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max = 0, count = 0;
        for(int i = 0; i < nums.length; i++) {
            if(nums[i] == 1) {
                count += 1;
                if (count > max) {
                    max = count;
                }
            } else {
                count = 0;
            }
        }
        return max;
    }
}

File: P485 Max Consecutive Ones/Solution.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        max, count = 0, 0
        for i in range(len(nums)):
            if nums[i] == 1:
                count += 1
                if count > max:
                    max = count
            else:
                count = 0
        return max